tests/e2e/set-ffi-env-e2e

```diff
@@ -261,13 +261,15 @@ fi
 echo
 info_message "Checking if QM already installed"
 info_message "=============================="
+QM_INST="$(rpm -qa qm)"
 QM_STATUS="$(systemctl is-enabled qm 2>&1)"
```

Comment:
When `qm` is not installed in the environment, error will occur when running to this line